beta/vercel.json

```diff
@@ -1,46 +1,253 @@
 {
-  "github": { "silent": true },
+  "github": {
+    "silent": true
+  },
   "trailingSlash": false,
   "redirects": [
-    { "source": "/tips/controlled-input-null-value.html", "destination": "/docs/forms.html#controlled-input-null-value", "permanent": false },
```

Comment:
@gaearon  I dont think we need this beta site's vercel.json because we actually do not have any redirects related to the new beta site. Should we change destination to older site? or remove them?

Reply:
beta site will become main site someday so we're going to need them (but we'll revisit their content)